Assets/Modules/AdvancedMemory/AdvancedMemory.cs

```diff
@@ -550,16 +546,13 @@ private int GetDigit(char c)
     string TwitchHelpMessage = "Enter the Forget Me Not sequence with \"!{0} press 531820...\". The sequence length depends on how many modules were on the bomb. You may use spaces and commas in the digit sequence.";
     #pragma warning restore 0414
 
-    public void TwitchHandleForcedSolve() {
+    public IEnumerator TwitchHandleForcedSolve() {
         Debug.Log("[Forget Me Not #"+thisLoggingID+"] Module forcibly solved.");
-        forcedSolve = true;
-        StartCoroutine(Solver());
-    }
-
-    private IEnumerator Solver() {
-        while(Position < Solution.Length) {
+        while (!done) yield return true; //Waits for all other mods on the bomb to be solved, if FMN needs to be solved right away, executing the solve command again will do this
```

Comment:
This should not be included, the entire purpose of forcing a solve is in the name. Solves should be forced no matter what.